crates/astria-sequencer/src/service/mod.rs

```diff
@@ -7,3 +7,39 @@ pub(crate) use consensus::Consensus;
 pub(crate) use info::Info;
 pub(crate) use mempool::Mempool;
 pub(crate) use snapshot::Snapshot;
+use tendermint::abci::Code;
+
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct AbciCode(u32);
+
+#[rustfmt::skip]
+impl AbciCode {
```

Comment:
Moved this to its own block so that rustfmt does not reorder the associated consts.